dotfiles/.zshrc

```diff
@@ -4,20 +4,16 @@
 _after_command()
 {
     local exit_code=$?
-    [ -z "${__begin+.}" ] && return
+    [ -z "${__begin_window+.}" ] && return
     local last_command=$(history -n -1 2>/dev/null)
-    if PS1=$(COLUMNS=$COLUMNS custom-zsh-prompt "$last_command" $exit_code $__begin "$(__git_ps1 '   %s')")
-    then
-        ([ $__window -ne $(getactivewindow) ] && notify-send -i dialog-information "CLI Ready" "$last_command" &)
-    fi
-    unset __begin __window
+    PS1=$(COLUMNS=$COLUMNS PWD=$PWD custom-zsh-prompt "$last_command" $exit_code $=__begin_window "$(__git_ps1 '   %s')")
```

Comment:
I want to use the Windows-compiled binary for both WSL and MSYS2, but it won't have access to the Linux environment variable `PWD` in WSL. Pass it as a command line argument. Do that same for `COLUMNS` just to be consistent.